src/backend/distributed/commands/index.c

```diff
@@ -328,8 +328,13 @@ ExecuteFunctionOnEachTableIndex(Oid relationId, PGIndexProcessor pgIndexProcesso
 		HeapTuple indexTuple = SearchSysCache1(INDEXRELID, ObjectIdGetDatum(indexId));
 		if (!HeapTupleIsValid(indexTuple))
 		{
-			ereport(ERROR, (errmsg("cache lookup failed for index with oid %u",
-								   indexId)));
+			if (EnablePropagationWarnings)
```

Comment:
This is also changing behavior. Can you remind which test does it fix?

(I think the name of the GUC is relatively good, we should only change behavior for WARNINGs, not ERRORs. Because errors are changing behavior)

Reply:
It is left from the beginning of the PR. It is unnecessary again.